code/panels/UpdatePanel.php

```diff
@@ -83,28 +85,42 @@ public function getLatestSilverStripeVersion() {
 		return $latestVersion;
 	}
 
-	public function isCurrentSilverStripeVersion() {
-		if ($this->getCurrentSilverStripeVersion() && $this->getLatestSilverStripeVersion()) {
-			return $this->getCurrentSilverStripeVersion() >= $this->getLatestSilverStripeVersion();
+	public function isNewerSilverStripeVersion($versionNumber, $latestVersionNumber) {
+		if ($versionNumber == $latestVersionNumber) {
```

Comment:
Or I could remove that if check completely. But I think it's nice to have in there because it saves the effort of comparing every part of the string for no reason.